R/uvhydrograph-render.R

```diff
@@ -837,3 +846,34 @@ getCorrectionsPlotConfig <- function(corrections, plotStartDate, plotEndDate, la
   
   return(plotConfig)
 }
+
+#' Get Rating Shifts Plot Config
+#' @description Given rating shifts, some information about the plot to build, will return a named list of gsplot elements to call
+#' @param ratingShifts list of data objects relavant to plotting corrections data
+#' @param plotStartDate start date of this plot 
+#' @param plotEndDate end date of this plot
+#' @param label label of that these corrections are associated with
+#' @param limits list of lims for all of the data which will be on here
+#' @return named list of gsplot calls. The name is the plotting call to make, and it points to a list of config params for that call
+getRatingShiftsPlotConfig <- function(ratingShifts, plotStartDate, plotEndDate, label, 
+                                     limits) {
+  
+  if(isEmptyOrBlank(ratingShifts) || nrow(ratingShifts) <= 0) {
+    return(list())
+  }
+  styles <- getUvStyles()
+  
+  ratingShiftsPositions <- getCorrectionPositions(ratingShifts)
```

Comment:
These functions seem to work for ratingShifts too, should I revamp them to be named more generically? getCorrectionPositions(), parseCorrectionsLabelSpacing)_, and getCorrectionArrows()